config/config.yaml

```diff
@@ -1,9 +1,11 @@
 # config.yaml
 directories:
-  image_dir: /home/ubuntu/vqasynth_sample
   output_dir: /home/ubuntu/vqasynth_output
 
 arguments:
+  source_repo_id: remyxai/vqasynth_sample
+  target_repo_id: salma-remyx/vqasynth_sample_processed
+  image_col: "image"
```

Comment:
something more explicit or descriptive than image_col

Reply:
@smellslikeml do you want `image_data_field` or `image_data_column`